### PR TITLE
Improve UI

### PR DESCRIPTION
**Pull request by priyanshunayan**
Fixes #

### Checklist
- [x] My branch is up-to-date with upstream/develop branch.
- [x] Everything works and tested for Python 3.5.2 and above.

### Description

This PR improves the UI/UX of the Agent GUI

### CODE REVIEW

**Comment by priyanshunayan** [comment]
The UI is live on http://34.67.128.133:3000/

File: requirements.txt [hunk 1]
```diff
@@ -2,5 +2,5 @@ flask
 Flask-Cors
 -e git+https://github.com/RedisGraph/redisgraph-py#egg=redisgraph
 -e git+https://github.com/HTTP-APIs/hydra-python-core#egg=hydra_python_core
--e git+https://github.com/HTTP-APIs/hydra-python-agent#egg=hydra_agent
+-e git://github.com/priyanshunayan/hydra-python-agent.git@get-request#egg=hydra_agent
```
Comment (xadahiya) [inline requirements.txt:5]: Please don't change this. Requirements should always point to the main repo.

Comment (priyanshunayan) [reply requirements.txt:5]: I can revert the change if this PR is merged: https://github.com/HTTP-APIs/hydra-python-agent/pull/149

Comment (priyanshunayan) [reply requirements.txt]: Reverted.

**Pull request merged**